### PR TITLE
Organization page layout with data integrations

### PR DESCRIPTION
**Pull request by AllStackDev1**
Closes: #1536

Mobile:
Simulator Screen Recording - iPhone 14 - 2023-03-10 at 22 15 53

Mobile Dropdown
Simulator Screen Shot - iPhone 14 - 2023-03-15 at 13 24 39

Web:
ezgif-3-1f6e5f10aa

Web Dropdown
Screenshot 2023-03-15 at 1 25 16 PM

# Setup organization page layout

## Background

As a user, I can view a page for any organiziation I have available, which will show that organization’s list of repositories. I can also see options to search my repositories, sort them, and filter them. (Functionality does not need to work yet for these options, they just need to be visible.)

## Acceptance

- [x] 1 column layout
- [x] sub-header spanning both columns, showing book icon and repositories title - should be sticky on scroll
- [x] Top of main section - search / filter / sort component
- [x] Main section - card for all org repos

### CODE REVIEW

File: expo-zustand-styled-components/src/screens/Organization/Organization.styles.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  padding: 16px;
+  align-items: center;
+  flex-direction: row;
+`;
+
+export const TabNavContainer = styled.View`
+  padding-top: 8px;
+  padding-horizontal: 16px;
+  border-bottom-width: 1px;
+  border-bottom-color: #e1e4e8;
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/screens/Organization/Organization.styles.ts:30]: I believe this color exists in the colors object, if it doesn't you can update it so we can reuse it where appropriate.

What do you think?

**Review by hdJerry — CHANGES_REQUESTED** [review]

**Review by hdJerry — DISMISSED** [review]

File: expo-zustand-styled-components/src/components/FilterDropdown/DropdownOverlay.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const DropdownOverlay = ({onPress}) => {
+  return (
+    <TouchableOpacity
+      activeOpacity={1}
+      onPress={onPress}
+      style={{
+        ...styles.dropdownOverlay,
+        ...{
+          backgroundColor: 'rgba(0,0,0,0.4)',
+        },
```
Comment (Megio) [inline expo-zustand-styled-components/src/components/FilterDropdown/DropdownOverlay.tsx:13]: I don't understand why we need to spread an object in this way

File: expo-zustand-styled-components/src/components/FilterDropdown/DropdownOverlay.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+};
+
+export default DropdownOverlay;
+
+const styles = StyleSheet.create({
+  dropdownOverlay: {
+    width: '100%',
+    height: '100%',
+  },
+});
```
Comment (Megio) [inline expo-zustand-styled-components/src/components/FilterDropdown/DropdownOverlay.tsx:26]: We are using styled component, can we "convert" this into a styled component?

File: expo-zustand-styled-components/src/components/FilterDropdown/DropdownWindow.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    minWidth: 150,
+  },
+  shadow: {
+    shadowColor: '#000',
+    shadowOffset: { width: 0, height: 6 },
+    shadowOpacity: 0.1,
+    shadowRadius: 10,
+    elevation: 10,
+  },
+});
```
Comment (Megio) [inline expo-zustand-styled-components/src/components/FilterDropdown/DropdownWindow.tsx:30]: We are using styled component, can we "convert" these into some styled components?

File: expo-zustand-styled-components/src/hooks/useLayoutDropdown.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  const dropdownWindowStyle = useMemo(() => {
+    return {
+      ...{
+        borderTopWidth: 0,
+        overflow: 'hidden',
+      },
+      ...{
+        top: dropdownPY,
+        position: 'absolute',
+      },
```
Comment (Megio) [inline expo-zustand-styled-components/src/hooks/useLayoutDropdown.ts:36]: DOn't understand what we are spreading here

**Comment by hdJerry** [comment]
What's the essence of the modal? it's just a dropdown. There has to be a better way than the modal. For mobile it's okay not for desktop.

Can you explain the decision?

**Review by Megio — APPROVED** [review]
LGTM

**Pull request merged**